app/support/end_reservation_only.rb

```diff
@@ -0,0 +1,34 @@
+class EndReservationOnly
+  def perform
+    order_details.each do |od|
+      od.transaction do
+        expire_reservation(od)
+      end
+    end
+  end
+
+  private
+
+  def order_details
+    reservation_only_order_details
+  end
+
+  def reservation_only_order_details
+    OrderDetail.purchased_active_reservations
+      .merge(Reservation.in_progress)
```

Comment:
Hmm, I may need to stop by today for a second. I'm kind of confused  why we need to set actual_end_at time but not actual_start_at. Is that for pricing? 

If so, why can't we use reserve_start_at and reserve_end_at for pricing?